proto/buf/alpha/registry/v1alpha1/user.proto

```diff
@@ -91,6 +99,8 @@ message ListUsersRequest {
   // The first page is returned if this is empty.
   string page_token = 2;
   bool reverse = 3;
+  // request with unspecified user state will return users with both user state.
+  UserState user_state = 4;
```

Comment:
Should this be `user_state_filter` instead?

Reply:
sounds good, I guess we need to apply this to `CountUsersRequest` too?

Reply:
Yes